test/integration/next-image-new/default/test/index.test.ts

```diff
@@ -121,17 +121,20 @@ function runTests(mode) {
       const links = await browser.elementsByCss('link[rel=preload][as=image]')
       const entries = []
       for (const link of links) {
+        const fetchpriority = await link.getAttribute('fetchpriority')
```

Comment:
Should we add a test case for `app` as well?

Reply:
Yes, in fact we need nearly every test case for `pages` repeated for `app`.

I will do that in a different PR because it will be much more work.

Reply:
I created PR #47267